### PR TITLE
 Update application.html.slim

### PR DESCRIPTION
**Pull request by morrme**

### CODE REVIEW

File: apps/web/templates/application.html.slim [hunk 1]
```diff
@@ -6,8 +6,15 @@ html
...
     link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600&amp;subset=cyrillic" rel="stylesheet"
-
+     
+    meta[charset='utf-8']
     meta name="viewport" content="width=device-width, initial-scale=1.0"
+    meta 
+    meta name="twitter:image" content="#{image_url('header-logo.png')}"
+    meta property="og:image" content="#{image_url('header-logo.png')}"
+    meta name="og:url" 
+    meta name="og:description:
```
Comment (davydovanton) [inline apps/web/templates/application.html.slim]: missed `"` :)

Comment (morrme) [inline apps/web/templates/application.html.slim]: @davydovanton I am sorry, this PR was sent in error, I was saving to grab more details from your site.  Please see questions in the issue.

**Comment by morrme** [comment]
We still need:

- Image for App Icons 
- I have generated a site map, not sure where in the structure to place it/add ref to it
- Gzip (compression) needs to be enabled in the application

**Comment by davydovanton** [comment]
Excellent work, can I help you with something?

**Comment by morrme** [comment]
@davydovanton thanks! yes, do you have images that could be used for the App Icons? They ideally need to be square, but I can crop them as needed and make them the proper sizes.  Also , are you familiar with enabling gzip compression? I am not familiar with how to do it on Hanami.

**Comment by davydovanton** [comment]
Hey, sorry for my delay. I want to merge this changes and create a new issue for other three issues. Thanks for your work! 👍

**Pull request merged**